tests/test_drs.py

```diff
@@ -35,14 +41,30 @@ def get_doc(has_version=True, urls=list(), drs_list=0):
         doc["version"] = "1"
     if urls:
         doc["urls"] = urls
-    # if drs_list > 0:
-    #     ret = {"drs_objects": []}
-    #     for _ in range(drs_list):
-    #         ret["drs_objects"].append(doc)
-    #     return ret
+    if has_description:
+        doc["description"] = "A description"
+    if has_updated_time:
+        doc["updated_time"] = "2023-03-14T17:02:54"
+    if has_created_time:
+        doc["created_time"] = "2023-03-13T17:02:54"
+
     return doc
 
 
+def get_bundle(client, user, has_description=True):
+    docs = [get_doc(), get_doc()]
+    dids = []
+    for doc in docs:
+        res = client.post("/index/", json=doc, headers=user)
+        assert res.status_code == 200
+        dids.append(res.json["did"])
+    bundle = get_bundle_doc(bundles=dids)
+    if has_description:
+        bundle["description"] = "A description"
```

Comment:
maybe `get_bundle_doc` should just accept a description?

Reply:
Is there a reason we'd need to check an inputted description rather than just a generic string here?

Reply:
i'm not sure i understand your question, i meant that you could do `bundle = get_bundle_doc(bundles=dids, has_description=True)` or `bundle = get_bundle_doc(bundles=dids, description=xxx)` on L61 and let `get_bundle_doc` add the description

but it's just a detail